src/main/java/edu/rosehulman/quota/AddUserController.java

```diff
@@ -1,4 +1,32 @@
 package edu.rosehulman.quota;
 
-public class AddUserController {  
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class AddUserController implements Route {
+
+  @Override
+  public Object handle(Request request, Response response) throws Exception {
+    String partnerId = request.params(":partnerId");
+    String productId = request.params(":productId");
+    String userId = request.params(":userId");
+
+    try {
+      // Add the user to our database
+      boolean added = Database.getInstance().addUser(partnerId, productId, userId);
+      if (!added) {
+        response.status(500);
+        return response;
+      }
+    } catch(Exception e) {
+      response.status(500);
+      return response;
+    }
+
+    // Send the user to Shared
+//    Response sharedRes = SharedServiceClient.getInstance().addUser(partnerId, productId, userId);
```

Comment:
Don't comment this out; In your test, mock out the SharedServiceClient just like the database and make the addUser method return the same response object that you mocked for this handle method. Then add an assert that the two response objects are the same.